src/text.c

```diff
@@ -997,6 +997,11 @@ void set_weather_layer_text(char* text) {
     set_text(weather, weather_text);
 }
 
+void set_weather_layer_color(GColor8 color) {
```

Comment:
We could either wrap the `set_text_color` calls with `#if defined PBL_PLATFORM_APLITE` or set different colors for that case :)

Reply:
Good call; I can probably get that in tonight.

Also, it would be good to make the colors configurable via the app; I just didn't take the time to make that happen.